Wox.Infrastructure/StringMatcher.cs

```diff
@@ -99,9 +99,7 @@ public static MatchResult FuzzySearch(string query, string stringToCompare, Matc
                     RawScore = Math.Max(score, pinyinScore)
                 };
 
-                return result.Score > 0 ? 
```

Comment:
do we know if score > 0 is still a valid scenario? A bit wary of removing this here. will note this down, merge, and investigate with a fix if needed.